### ISSUE TITLE
[a11y] /contribute all page content should be contained in landmarks

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15340]
# Axe-core® Accessibility Results for /en-US/contribute/

Page URL: https://www-dev.allizom.org/en-US/contribute/

## axe-core found 6 violations

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | All page content should be contained by landmarks | region | Best practice | moderate | 6   |

### Failed

1. All page content should be contained by landmarks

[Learn more](https://dequeuniversity.com/rules/axe/4.10/region?application=playwright)

region

Best practice

Ensure all page content is contained by landmarks

moderate

Issue Tags: cat.keyboard best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-split<br>```<br><br>**Element source**<br><br>```<br><section class="mzp-c-split mzp-t-split-nospace"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 2   | **Element location**<br><br>```<br>.mzp-c-newsletter-header<br>```<br><br>**Element source**<br><br>```<br><header class="mzp-c-newsletter-header"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 3   | **Element location**<br><br>```<br>label[for="id_email"]<br>```<br><br>**Element source**<br><br>```<br><label for="id_email">Your email address:</label><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 4   | **Element location**<br><br>```<br>#id_email<br>```<br><br>**Element source**<br><br>```<br><input type="email" name="email" required="" data-testid="newsletter-email-input" maxlength="320" placeholder="yourname@example.com" class="mzp-js-email-field" id="id_email"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 5   | **Element location**<br><br>```<br>.mzp-c-fieldnote<br>```<br><br>**Element source**<br><br>```<br><span class="mzp-c-fieldnote"><br>              <br>              <br>                We will only send you Mozilla-related information. You can unsubscribe at any time.<br>              <br>            </span><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |
| 6   | **Element location**<br><br>```<br>.contribute-banner-wrapper<br>```<br><br>**Element source**<br><br>```<br><section class="contribute-banner-wrapper"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |

**Comment by alexgibson** [comment]
Tip: This can probably be fixed by wrapping the core page content in a `<main>` element, and then making the last three sections (newsletter, get help, problems with a web page) `<aside>` landmarks.

**Comment by clarapenn** [comment]
@stevejalim 👋
